plugins/angular.js

```diff
@@ -65,3 +86,5 @@ angularPlugin._normalizeData = function (data) {
 };
 
 module.exports = angularPlugin;
+// Install plugin when included
+angularPlugin(window.Raven);
```

Comment:
Uh, I mean, it basically just can't go here. We'll have to come up with something else.

I think one big issue is: Raven should really be configured before you even initialize Angular. That way it can pick up an error in the Angular initialize process. I understand it's convention to pipe your configuration through Angular, but ... it makes this all really horrible.